src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab
+    = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
```

Comment:
you can get `selectedTab` as prop from parent component here instead of `selectedTabId` and do not repeat yourself

Reply:
Not fixed 😢

Reply:
> you can get `selectedTab` as prop from parent component here instead of `selectedTabId` and do not repeat yourself

Still not fixed 🤷‍♂️